backend/services/src/github_services/utilities.bal

```diff
@@ -42,11 +45,31 @@ public function checkLabel(string labelName) returns @untainted string[] {
     }
 }
 
-# The `createLabel` function will create a label in a specified git repository.
-#
-# + labelName        - Name of the label.
-# + labelDescription - Description of the label.
-# + return           - Returns a **string[]** which indicates the status.
+
+public function checkLabels(string labelName) returns @untainted string[] {
```

Comment:
Whats the purpose of this method? It just returns an array.

Reply:
This method return whether the label is available and the string containing returned status code and the message

Reply:
It's not implemented is it?